README.md

```diff
@@ -180,6 +182,61 @@ OK
 ```
 
 
+### `cf create-autoscaling-credential`
+
+Create custom metric credential for an application, the credential will be displayed in JSON format. Random credential pair will be created if username and password are not specified.
```

Comment:
Credential is required when submitting custom metrics to app-autoscaler.  
If the application is connected to autoscaler through a service binding,  the required credential could be found in Cloud Foundry VCAP_SERVICES environment variables.  
This command is used to generate autoscaler credential and display it in JSON format. Then you need to set this credential to your application through environment variable or user-provide-service.
Random credential pair will be created by default when username and password are not specified by --username and --password option.